discovery-provider/src/schemas/track_schema.json

```diff
@@ -454,6 +457,25 @@
                 "tip_user_id"
             ],
             "title": "PremiumConditionsTipUserId"
+        },
+        "PremiumConditionsUSDCPurchase": {
```

Comment:
@isaacsolo looks like we don't use this schema.json anymore, do we have plans to use it again or should I remove it?